cypress/integration/createBoxesAndProducts.js

```diff
@@ -0,0 +1,106 @@
+import uuidv4 from "uuid/v4";
+
+describe('Create and delete boxes', function() {
+    let testUserMail;
+    let testPwd;
+
+    before(function() {
+        cy.getTestData().then(($result) => {
+            testUserMail = $result.testUserMail;
+            testPwd = $result.testPwd;
+        });
+    })
+
+    beforeEach(function() {        
+        cy.reLogin(testUserMail, testPwd);
+    });
+
+    afterEach(function() {
+        cy.wait(2000);
+    });
+
+    it('Create new product', () => {
+        let productName = uuidv4().substring(0,6);
+        cy.navigateToProductsPage();
+        cy.get('button[data-cy=addProductButton]').click();
+        cy.get('#select-category').click();
+        cy.get('li[tabindex=0]').click();
```

Comment:
First item in the drop down list or something? Might need to narrow down this selector

Reply:
Exactly. There's just one dropdown list in the form so I thought it's enough. Or you don't like that he always takes the first element? I should be able to make it random pretty easy if it's encouraged. I just thought it doesn't make much of a difference for the test.

Reply:
Ok, changed this to `cy.get('li[id=category]').first()` after reading through Cypress documentation